main.tf

```diff
@@ -43,26 +50,24 @@ resource "google_compute_target_http_proxy" "default" {
   project = var.project
   count   = var.http_forward ? 1 : 0
   name    = "${var.name}-http-proxy"
-  url_map = compact(
-  concat([
-    var.url_map], google_compute_url_map.default.*.self_link),
-  )[0]
+  url_map = local.url_map
 }
 
 # HTTPS proxy  when ssl is true
 resource "google_compute_target_https_proxy" "default" {
-  project          = var.project
-  count            = var.ssl ? 1 : 0
-  name             = "${var.name}-https-proxy"
-  url_map          = compact(
-  concat([
-    var.url_map], google_compute_url_map.default.*.self_link), )[0]
+  project = var.project
+  count   = var.ssl ? 1 : 0
+  name    = "${var.name}-https-proxy"
+  url_map = local.url_map
+
   ssl_certificates = compact(concat(var.ssl_certificates, google_compute_ssl_certificate.default.*.self_link, ), )
+  ssl_policy       = var.ssl_policy
+  quic_override    = var.quic ? "ENABLE" : "NONE"
 }
 
 resource "google_compute_ssl_certificate" "default" {
   project     = var.project
-  count       = var.ssl && !var.use_ssl_certificates ? 1 : 0
+  count       = var.ssl && ! var.use_ssl_certificates ? 1 : 0
```

Comment:
Nit: remove space for consistency

Reply:
Fixed although `terraform fmt` does re-introduce the whitespace, so we should keep it?

Reply:
Huh, TIL. Sure go ahead and keep it then :)